.circleci/docker/build.sh

```diff
@@ -185,6 +185,13 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
+  pytorch-linux-focal-py3-clang7-tsan)
+    ANACONDA_PYTHON_VERSION=3.7
+    CLANG_VERSION=7
+    PROTOBUF=yes
+    DB=yes
+    VISION=yes
+    ;;
```

Comment:
Why do we need to have a separate config here? I.e. ASAN and TSAN can share the same image, can't they?

Reply:
True, this is exactly the same docker image as ASAN.  Let me get rid of this.